app/helpers/claims_helper.rb

```diff
@@ -40,4 +40,9 @@ def messaging_permitted?(message)
     end
     false
   end
+
+  def miscellaneous_fees_notice(claim)
```

Comment:
naming wise i think something like `page_notice_for` would be clearer here. 🤔 and possibly be more specific generally, so `unused_material_notice` as that is, currently, its sole purpose, and may never change.